config/webpack.config.web.js

```diff
@@ -49,6 +49,7 @@ async function makeConfig(
     },
     devServer: {
       inline: false,
+      host: mode == "development" ? '0.0.0.0' : 'localhost',
```

Comment:
I would avoid this and use explicit setting of host instead. The dev server isn't particularly secure and accepting any host by default may cause unnecessary risk for people using this on their machines directly instead of from a container.

Reply:
now default `0.0.0.0` only on `docker-compose`, otherwise it's `localhost`